.env.example

```diff
@@ -3,17 +3,24 @@
 # Network to use in config
 NETWORK=
 
+#############################################################################
```

Comment:
i ahve some values for this file in my local copy but you'll need to set up your own api keys, etc.